Makefile

```diff
@@ -1,4 +1,10 @@
-data/whitelisted_ext_idefs.json: 
+# Putting the top one first so "make" without argument picks up the last result
+data/references.jsonl: \
+	data/matched_references.jsonl \
+	data/matched_item_references.jsonl
+	cat $^ > $@
```

Comment:
Great idea, thanks!

Also general question (unrelated to this PR), I noticed that make tries to run all of the scripts even if the file is already present in the data folder. Is it the normal way it should work? Am I missing a parameter here?

Reply:
if the file exists, it should just skip it.

Reply:
Weirdly enough it doesn't work that way for me :/

Reply:
That's strange, that's how I do it in my localhost and the labs instance (by moving files around).
Maybe that's the problem I encountered before. If A depends on B and B doesn't exists but A does. It still re-runs A (after running B) because it says "the dependency has changed" I guess it's so linuxy and systemdy :D